Control Portal/control-portal-navigation-update.md

```diff
@@ -1,32 +1,14 @@
-{{{
-  "title": "Control Portal Navigation Update",
-  "date": "7-6-2016",
-  "author": "Nathan Young",
-  "attachments": [],
-  "related-products" : [],
-  "contentIsHTML": false,
-  "sticky": false
-}}}
-
```

Comment:
Please do not remove this section. Please update with your information and the most recent date of change.